test/test_distributed.py

```diff
@@ -645,7 +645,7 @@ def _test_broadcast_helper(
                         tensor = tensor.cuda(rank_to_GPU[rank][0])
                     dist.broadcast(tensor, src, group_id)
                     self.assertEqual(tensor.size(), expected_tensor.size())
-                    self.assertEqual(tensor.ne(expected_tensor).max(), 0)
+                    self.assertEqual(tensor.ne(expected_tensor).max(), torch.tensor(False, dtype=torch.bool))
```

Comment:
I feel like passing `False` here should be OK. But I guess this is testing code so it doesn't matter too much.